pom.xml

```diff
@@ -199,13 +191,6 @@
                 </exclusion>
             </exclusions>
         </dependency>
-
-        <dependency>
```

Comment:
Is it OK to remove this? We still run tests against JDK 8 right?

Reply:
it should be needed for > 8, but we don't need it anymore, likely because we have it as a transitive dependency.